src/components/ProjectSection/ProjectData.ts

```diff
@@ -51,126 +53,142 @@ export const PastProjects: Project[] = [
   },
   {
-    id: 5,
+    id: 4,
     name: "TechstArcade",
     github: "https://github.com/techstartucalgary/techstarcade",
     pmName: "Madeline Mazurek",
     website: website,
     image: techstarcade,
+    year: "",
     description: "Retro style mini games in a arcade style chrome extension.",
   },
   {
-    id: 6,
+    id: 5,
     name: "NoteAId",
     github: "https://github.com/Tech-Start-UCalgary/NoteAId",
     website: website,
     image: noteAid,
     pmName: "Mahdi Varposhti",
+    year: "",
     description:
       "A mobile app that helps students with studying for exams by summarizing their handwritten notes",
   },
   {
-    id: 7,
+    id: 6,
     name: "Where To?",
     github: "https://github.com/Tech-Start-UCalgary/decision-frontend",
     website: website,
     image: whereTo,
     pmName: "Vivian Huynh",
+    year: "",
     description:
       "An app that helps groups make decisions when planning a social gathering by voting",
   },
   {
-    id: 8,
-    name: "Aquavolution",
-    github: "https://github.com/Tech-Start-UCalgary/Aquavolution",
-    website: website,
-    image: aquavolution,
-    pmName: "Elgiz Abbasov",
-    description:
-      "Born a tiny fish, try to survive among other predators, be careful of the human waste in the ocean though!",
-  },
-  {
-    id: 9,
+    id: 7,
     name: "Snowbot",
     github: "",
     website: website,
     image: snowbot,
     pmName: "Ali Siddiqi",
+    year: "",
     description: "A robot that autonomously clears snow on sidewalks",
   },
   {
-    id: 10,
+    id: 8,
     name: "OneApp",
     github: "https://github.com/glassthunder/TECHSTART",
     pmName: "Felix Vaughan",
     website: website,
     image: oneHub,
+    year: "",
     description:
       "A hub for your social media that shows you info and stats from your favorite sites. Switching apps is so 2020!",
   },
   {
-    id: 11,
+    id: 9,
     name: "SupplyMe",
     github: "https://github.com/Tech-Start-UCalgary/SupplyMe",
     pmName: "Darryl Huët",
     website: website,
     image: supplyMe,
+    year: "",
     description:
       "An app that uses the Ethereum blockchain with QR Codes to track package and parcel transfers",
   },
   {
-    id: 12,
+    id: 10,
     name: "Resume Tracker",
     github: "",
     pmName: "Daniel Rashidian",
     website: website,
     image: resumeTracker,
+    year: "",
     description:
       "Resume Tracker rates your resume and provides feedback on how to improve it",
   },
   {
-    id: 13,
+    id: 11,
     name: "TSU Website",
     github: "https://github.com/Tech-Start-UCalgary/tsu-website",
     pmName: "Joel Happ, Niyousha Raeesinejad",
     website: website,
     image: techStartWebsite,
+    year: "",
     description:
       "We built this website from scratch with React, TypeScript, and a Django backend!",
   },
 ];
 
 export const CurrentProjects: Project[] = [];
+
+export const TopProjects: Project[] = [
```

Comment:
I don't think you need a new array for the featured projects. Instead I suggest adding a boolean field `featured` to the `Project` type and iterate over the `PastProjects` to filter the featured projects to display in the featured section. This will ensure all our past projects are included at the bottom of the page as well.